src/services/datahub-client.ts

```diff
@@ -140,16 +142,22 @@ export class HttpDatahubClient implements DatahubClient {
   public async getPagesRankingByPeriod({
     domainName,
     dateFrom,
+    pageSize,
+    pageNumber,
   }: {
-    domainName: number;
+    domainName: string;
     dateFrom: Date;
+    pageSize: number;
+    pageNumber: number;
   }): Promise<PageRankingResult> {
     try {
       const response = await this.customerGet<any>(
         `domains/${domainName}/events/summarized-by-page`,
         {
           startDate: dateFrom.toISOString(),
           sortBy: 'visitors',
+          pageSize: pageSize || '',
+          pageNumber: pageNumber || '',
```

Comment:
These values should not be numbers?

```suggestion
          pageNumber: pageNumber || 0,
```

Or

```suggestion
          pageNumber: pageNumber || null,
```